bot/exts/events/hacktoberfest/_utils.py

```diff
@@ -0,0 +1,452 @@
+
+    PRs with an 'invalid' or 'spam' label are ignored unless merged or approved.
+
+    PRs have to be in a repository that has a 'hacktoberfest' topic,
+    unless the PR is labelled 'hacktoberfest-accepted' for it to count.
+
+    If a valid `github_username` is provided, an embed is generated and posted to the channel.
+
+    Otherwise, a helpful error message is posted.
+    """
+    async with ctx.typing():
+        prs = await get_october_prs(ctx.bot, github_username)
+
+        if prs is None:  # Will be `None` if the user was not found
+            await ctx.send(
+                embed=discord.Embed(
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description=f"GitHub user `{github_username}` was not found.",
+                    colour=discord.Colour.red()
+                )
+            )
+            return
+
+        if prs:
+            stats_embed = await build_stats_embed(ctx.bot, github_username, prs)
+            await ctx.send("Here are some stats!", embed=stats_embed)
+        else:
+            await ctx.send(f"No valid Hacktoberfest PRs found for '{github_username}'")
+
+
+async def build_stats_embed(bot: Bot, github_username: str, prs: list[dict]) -> discord.Embed:
+    """Return a stats embed built from github_username's PRs."""
+    logging.info(f"Building Hacktoberfest embed for GitHub user: '{github_username}'")
+    in_review, accepted = await categorize_prs(bot, prs)
+
+    n = len(accepted) + len(in_review)  # Total number of PRs
+    if n >= PRS_FOR_SHIRT:
+        shirtstr = f"**{github_username} is eligible for hacktoberfest swag!**"
+    elif (remaining_prs := PRS_FOR_SHIRT - n) == 1:
+        shirtstr = f"**{github_username} is 1 PR away from being eligible for hacktoberfest swag!**"
+    else:
+        shirtstr = f"**{github_username} is {remaining_prs} PRs away from being eligible for hacktoberfest swag!**"
+
+    stats_embed = discord.Embed(
+        title=f"{github_username}'s Hacktoberfest",
+        color=Colours.purple,
+        description=f"{github_username} has made **{n}** valid {contributionator(n)} in October.\n\n{shirtstr}\n\n"
+    )
+
+    stats_embed.set_thumbnail(url=f"https://www.github.com/{github_username}.png")
+    stats_embed.set_author(
+        name="Hacktoberfest",
+        url="https://hacktoberfest.digitalocean.com",
+        icon_url="https://avatars1.githubusercontent.com/u/35706162?s=200&v=4"
+    )
+
+    # This will handle when no PRs in_review or accepted
+    review_str = build_prs_string(in_review, github_username) or "None"
+    accepted_str = build_prs_string(accepted, github_username) or "None"
+    stats_embed.add_field(
+        name=":clock1: In Review",
+        value=review_str
+    )
+    stats_embed.add_field(
+        name=":tada: Accepted",
+        value=accepted_str
+    )
+
+    logging.info(f"Hacktoberfest PR built for GitHub user '{github_username}'")
```

Comment:
```suggestion
    logging.info(f"Hacktoberfest embed built for GitHub user '{github_username}'")
```
So it looks similar to line 111